content/6_Advanced/MinCostFlow.problems.json

```diff
@@ -49,8 +49,7 @@
       "isStarred": false,
       "tags": [],
       "solutionMetadata": {
-        "kind": "link",
-        "url": "https://codeforces.com/blog/entry/74493?#comment-586020"
+        "kind": "internal"
```

Comment:
I think you should leave this as it was before because your solution doesn't use min cost flow

Reply:
oops i added this. removing this breaks the deployment though.

Reply:
Huh that's weird

Reply:
@thecodingwizard @jeffreymeng any idea why this happens?

Reply:
This is intended behavior:
> We assume that if there is an internal solution, we should use it; therefore, the build will throw an error if there is an internal solution but the `solutionMetadata`'s `kind` isn't set to `'internal'`.

https://usaco.guide/general/working-mdx/#problem-lists

Reply:
It's intended. Maybe just add this link to the internal solution?

Reply:
So how should I fix it, or should I just leave it as-is?

Reply:
Add a "solution 2" thing under the official solution and link the cf comment for now